packages/node/src/policy-gen/generate.js

```diff
@@ -109,44 +178,80 @@ const ABS_POLICY_PATH = nodePath.resolve(DEFAULT_POLICY_PATH)
  * Generates a LavaMoat policy or debug policy from a given entry point using
  * `@endo/compartment-mapper`
  *
- * @param {string | URL} entrypointPath
+ * @param {string | URL} entrypoint
  * @param {GeneratePolicyOptions} [opts]
  * @returns {Promise<LavaMoatPolicy>}
  * @public
  */
 export const generatePolicy = async (
-  entrypointPath,
+  entrypoint,
   {
-    policyDebugPath = ABS_POLICY_DEBUG_PATH,
-    policyPath = ABS_POLICY_PATH,
+    policyDebugPath: rawPolicyDebugPath,
+    policyPath: rawPolicyPath = ABS_POLICY_PATH,
+    policyOverridePath: rawPolicyOverridePath,
+    policyOverride,
     writableFs = nodeFs,
     readPowers = defaultReadPowers,
-    isAbsolute = nodePath.isAbsolute,
+    isAbsolute = defaultIsAbsolute,
     write: shouldWrite = false,
     debug,
+    readFile = nodeFs.promises.readFile,
     log = defaultLog,
+    trustRoot = DEFAULT_TRUST_ROOT_COMPARTMENT,
     ...generateOpts
   } = {}
 ) => {
   await Promise.resolve()
 
-  if (entrypointPath instanceof URL) {
-    entrypointPath = readPowers.fileURLToPath(entrypointPath)
-  }
+  const entrypointPath = toPath(entrypoint)
+
   assert(
     isAbsolute(entrypointPath),
-    `entrypointPath must be an absolute path; got ${entrypointPath}`
+    `entrypoint must be an absolute path; got ${entrypointPath}`
   )
-  if (shouldWrite) {
-    assert(
-      isAbsolute(policyPath),
-      `policyPath must be an absolute path; got ${policyPath}`
-    )
-    if (policyDebugPath) {
+
+  const policyPath = toPath(rawPolicyPath)
+  const policyDir = nodePath.dirname(policyPath)
+
+  assert(
+    isAbsolute(policyPath),
+    `policyPath must be an absolute path; got ${policyPath}`
```

Comment:
AFAIR That's not true for the user, we're resolving them from rootDir. This error would be misleading